packages/api/src/external/hie/tally-doc-query-progress-with-source.ts

```diff
@@ -0,0 +1,119 @@
+import { DocumentQueryProgress, Progress } from "@metriport/core/domain/document-query";
+import { getStatusFromProgress } from "@metriport/core/domain/document-query";
+import { MedicalDataSource } from "@metriport/core/external/index";
+import { PatientExternalData } from "@metriport/core/domain//patient";
+import { ProgressType } from "@metriport/core/domain/document-query";
+import { Patient } from "@metriport/core/domain/patient";
+import { PatientModel } from "../../models/medical/patient";
+import { executeOnDBTx } from "../../models/transaction-wrapper";
+import { getPatientOrFail } from "../../command/medical/patient/get-patient";
+import { handleWebhookBeingSent } from "../../command/medical/document/process-doc-query-webhook";
+import { aggregateAndSetHIEProgresses, HIEPatientData } from "./set-doc-query-progress-with-source";
+
+type DynamicProgress = Pick<Progress, "successful" | "errors">;
+
+export type TallyDocQueryProgressWithSource = {
```

Comment:
nit: I feel like the type should be something like DocQueryProgressWithSourceTally or something that is a non phrase as opposed to a verb phrase like the function

Reply:
Its in line with the function name `tallyDocQueryProgressWithSource` i could add Params at the end but we dont have that logic anywhere